src/herder/TransactionQueue.cpp

```diff
@@ -658,6 +661,8 @@ TransactionQueue::removeApplied(Transactions const& appliedTxs)
     {
         auto& bannedFront = mBannedTransactions.front();
         bannedFront.emplace(h);
+        CLOG_DEBUG(Overlay, "Ban transaction {}", hexAbbrev(h));
```

Comment:
nit: what do you think about specifying that it's applied, like "Ban applied transaction {}"?

Reply:
Updated!